randovania/data/json_data/prime1.txt

```diff
@@ -2403,582 +2411,1261 @@ Ruins Entrance
+      Morph Ball Bomb and Morph Ball and Spider Ball and After Unlock Sun Tower spider track
+
+> Event - Unlock spider track; Heals? False
+  * Event Unlock Sun Tower spider track
+  > Dock to Transport to Magmoor Caverns North
+      Trivial
+
+> Event - Activate Sunchamber Chozo Ghosts layer; Heals? False
+  * Event Activate Sunchamber Chozo Ghosts layer
+  > Dock to Transport to Magmoor Caverns North
+      Trivial
 
 ----------------
 Transport Access North
 Asset id: 986845711
 > Dock to Hive Totem; Heals? False
-  * Normal Door to Hive Totem/Dock to Transport Access North
+  * Missile Blast Shield to Hive Totem/Dock to Transport Access North
+  > Dock to Transport to Magmoor Caverns North
+      Morph Ball
+  > Pickup (Energy Tank)
+      Trivial
 
 > Dock to Transport to Magmoor Caverns North; Heals? False
   * Normal Door to Transport to Magmoor Caverns North/Dock to Transport Access North
+  > Dock to Hive Totem
+      Morph Ball Bomb and Morph Ball
+
+> Pickup (Energy Tank); Heals? True
+  * Pickup 17; Major Location? False
+  > Dock to Hive Totem
+      Trivial
 
 ----------------
 Sunchamber Access
 Asset id: 1422133653
 > Dock to Sunchamber; Heals? False
   * Normal Door to Sunchamber/Dock to Sunchamber Access
+  > Dock to Sunchamber Lobby
+      Trivial
 
 > Dock to Sunchamber Lobby; Heals? False
   * Normal Door to Sunchamber Lobby/Dock to Sunchamber Access
+  > Dock to Sunchamber
+      Trivial
 
 ----------------
 Gathering Hall
 Asset id: 1206336453
 > Dock to Watery Hall Access; Heals? False
   * Normal Door to Watery Hall Access/Dock to Gathering Hall
+  > Dock to Gathering Hall Access
+      Trivial
 
 > Dock to Gathering Hall Access; Heals? False
   * Normal Door to Gathering Hall Access/Dock to Gathering Hall
+  > Dock to Watery Hall Access
+      Trivial
+  > Dock to Save Station 2
+      Trivial
+  > Dock to East Atrium
+      Morph Ball
 
 > Dock to Save Station 2; Heals? False
-  * Normal Door to Save Station 2/Dock to Gathering Hall
+  * Missile Blast Shield to Save Station 2/Dock to Gathering Hall
+  > Dock to Gathering Hall Access
+      Trivial
 
 > Dock to East Atrium; Heals? False
   * Normal Door to East Atrium/Dock to Gathering Hall
+  > Pickup (Missile Expansion)
+      Morph Ball Bomb and Space Jump Boots and Morph Ball
+
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 18; Major Location? False
+  > Dock to East Atrium
+      Trivial
 
 ----------------
 Totem Access
 Asset id: 3934929011
 > Dock to Ruined Gallery; Heals? False
   * Normal Door to Ruined Gallery/Dock to Totem Access
+  > Dock to Hive Totem
+      Trivial
 
 > Dock to Hive Totem; Heals? False
   * Normal Door to Hive Totem/Dock to Totem Access
+  > Dock to Ruined Gallery
+      Trivial
 
 ----------------
 Map Station
 Asset id: 442183190
 > Dock to Ruined Gallery; Heals? False
-  * Normal Door to Ruined Gallery/Dock to Map Station
+  * Missile Blast Shield to Ruined Gallery/Dock to Map Station
```

Comment:
Might as well add connections for this. New node "Map Station", trivial for both connections

Reply:
Done